resources/content/schema/user/user-auth.json

```diff
@@ -80,19 +109,29 @@
         "nic" : "r",
         "nic.macAddress" : "r",
 
-        "schema" : "r",
-
+        "schema.*" : "r",
+        
+        "statsAccess.*" : "r",
+      
         "subnet" : "r",
 
         "subscribe" : "cr",
         "subscribe.eventNames" : "cr",
 
         "port" : "ru",
+        "port.accountId" : "r",
+        "port.instanceId" : "r",
         "port.publicPort" : "ru",
+        "port.privateIpAddressId": "r",
         "port.privatePort" : "r",
         "port.publicIpAddressId" : "r",
         "port.protocol" : "r",
-
+        
+        "processExecution" : "r",
+        "processExecution.processInstanceId" : "",
+        
+        "processInstance" : "r",
```

Comment:
Do you not have a test for this one for the user role? I think it would have caught the fact that Id fields are readable.